### PR TITLE
Add possibility to pass all dag's parameters during converting from workflow

### PR DESCRIPTION
**Pull request by faderskd**

### CODE REVIEW

File: biggerquery/deployment.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def job_callable(**kwargs):
     return job_callable
 
 
-def workflow_to_dag(workflow, start_from, dag_id):
+def workflow_to_dag(workflow, start_from, dag_id, **dag_kwargs):
```
Comment (beczkowb) [inline biggerquery/deployment.py]: Adding the kwargs here is not enough, you need to allow the user to pass the kwargs to the Workflow class and then use the kwargs inside workflow_to_dag

Comment (faderskd) [reply biggerquery/deployment.py]: done

Comment (beczkowb) [reply biggerquery/deployment.py]: Did you test it using Cloud Composer?

Comment (faderskd) [reply biggerquery/deployment.py]: not yet, today will be tested

Comment (faderskd) [reply biggerquery/deployment.py]: after that it will be ready to merge I guess

**Pull request merged**